### DISCUSSION TITLE
Stream picture with audio source

### DISCUSSION THREAD

**Discussion by db-doula** [datarhei/restreamer#528] [Q&A] [answered]
Hello,

I want to ask how to stream a picture with audio source? if anyone have a tuto or an updated code.

Thanks for the help

**Comment by svenerbeck** [comment, marked as answer]
Hello @db-doula 

Out of the box, this is not possible. You would have to make some adjustments directly in the core of FFmpeg. However, it is possible, in principle, to achieve this. For example, this user managed to create a short video loop with an audio source: [https://www.youtube.com/watch?v=e8FGXIJNIV4](https://www.youtube.com/watch?v=e8FGXIJNIV4)

I will include this as a feature request. The feature is quite cool and can be helpful for various applications.

Cheerio
Sven

**Comment by svenerbeck** [comment]
We have converted the request into a feature request. Follow the discussion  https://github.com/datarhei/restreamer/discussions/345
 to stay current and see when your feature becomes active.  

Please appreciate it if we only implement some features, which may take some time. 

Cheerio
Sven
